pkg/reconciler/v1alpha1/service/resources/route.go

```diff
@@ -37,6 +45,7 @@ func MakeRoute(service *v1alpha1.Service) (*v1alpha1.Route, error) {
 			OwnerReferences: []metav1.OwnerReference{
 				*kmeta.NewControllerRef(service),
 			},
+			Annotations: annotations(service),
```

Comment:
just use pkg/resources.FilterMap.
Also do you want _that_ single annotation and filter everything else?

Reply:
switched to `resources.FilterMap` function, thanks
I don't see any other service annotations that may be needed in the route, so I filter everything else

Reply:
Well, if route is owned by a service it would inherit service annotations. @mattmoor  -- do we care?

Reply:
we may pass service annotations to the route, but I'd definitely filter some of them out, e.g. service's last applied configuration value
and for the consistency - shouldn't revisions and configurations also inherit service annotations?

Reply:
In general I think we propagate most of the annotations/labels.

Reply:
Updated. We didn't get a consensus on this question, so I agreed to propagate all service annotations to the Route :slightly_smiling_face: